Pipelines/test-execution-steps.yml

```diff
@@ -0,0 +1,122 @@
+parameters:
+  - name: IsMultiRegionEnabled
+    type: boolean
+    default: false
+
+  - name: RegionNumber
+    type: number
+
+  - name: IsClusterRequired
+    type: boolean
+    default: false
+
+  - name: JmeterFolderPath
+    type: string
+    default: JMeterFiles
+
+  - name: JmeterFileName
+    type: string
+    default: SampleTestPlan.jmx
+
+  - name: Threads
+    type: number
+    default: 10
+
+  - name: Duration
+    type: number
+    default: 300
+
+  - name: Loops
+    type: number
+    default: 3
+
+  - name: RampUpTime
+    type: number
+    default: 10
+
+  - name: Tenant
+    type: string
+
+  - name: Namespace
+    type: string
+
+  - name: ServiceConnection
+    type: string
+
+  - name: KeyVaultName
+    type: string
+
+  - name: SecretNames
+    type: string
+
+  - name: AKSRegion
+    type: string
+
+  - name: AKSResourceGroup
+    type: string
+
+  - name: AKSClusterName
+    type: string
+
+  - name: AKSSPNClientId
+    type: string
+
+  - name: PerfTestResourceId
+    type: string
+
+  - name: PerfTestClientId
+    type: string
+
+  - name: CSVFileNames
```

Comment:
are we passing multiple files here ?

Reply:
not sure, we could.. it's the existing functionality